### PR TITLE
Use GQL API for chalmrest

### PR DESCRIPTION
**Pull request by lindskogen**
Make lunch fetching API use GQL

### CODE REVIEW

**Review by Vilddjur — APPROVED** [review]
This fixes the broken api calls aswell.
Get this to production digIT so I can read the lunch menu again.

**Review by Hoidi — APPROVED** [review]
This looks good

File: chalmrest.rb [hunk 1]
```diff
@@ -37,65 +35,81 @@ def meals
...
+          location: restaurant[:location]
+        }
+      end
     end
   end
 
-  def fetch_meals_for_id(id)
-    url = build_chalmrest_url(id)
+  def fetch_restaurant_by_id(http, id)
+    searchDate = "2019-10-21"
```
Comment (Edholm) [inline chalmrest.rb]: ska denna vara hårdkodad?

File: chalmrest.rb [hunk 2]
```diff
@@ -47,7 +47,7 @@ def fetch_data
   end
 
   def fetch_restaurant_by_id(http, id)
-    searchDate = "2019-10-21"
+    searchDate = Date.today.to_s
```
Comment (Edholm) [inline chalmrest.rb:50]: nvm

**Pull request merged**